packages/react-components/tests/log-management/log-management.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { render, waitFor } from '@testing-library/react';
+import React from 'react';
+import { LogManagement } from '../../lib';
+
+const getLogLabels = () => [
+  { label: 'Web Server', value: 'web-server' },
+  { label: 'RMF core', value: 'rmf-core' },
+];
+
+const getLogs = () => {
+  const rows = [];
+  for (let i = 0; i < 200; i++) {
+    rows.push({
+      message: 'Test' + i,
+      level: 'WARN',
+      timestamp: 'Mon Jan  1 00:00:02 UTC 2001',
+    });
+  }
+  return rows;
+};
+
+const getLogsPromise = async () => await getLogs();
+const getLabelsPromise = async () => await getLogLabels();
+
+it('smoke test', async () => {
+  // Added the waitFor because this component is updating a state inside a useEffect.
+  await waitFor(() => {
+    render(<LogManagement getLabels={getLabelsPromise} getLogs={getLogsPromise} />);
+  });
```

Comment:
Would it be better to use `act`?

Reply:
Actually, from what I [read](https://github.com/testing-library/react-testing-library/issues/535#issuecomment-660267339), the RTL render function is already wrapped in an act by default. In this case, we need the waitFor because we need to wait until the `useEffect` async content is executed. 

https://testing-library.com/docs/dom-testing-library/api-async/#waitfor

Offtopic: during my search about the problem I found this interesting post https://kentcdodds.com/blog/common-mistakes-with-react-testing-library =)

Reply:
thanks for the explaination